embc-app/Controllers/VolunteersController.cs

```diff
@@ -49,7 +49,12 @@ public async Task<IActionResult> Get(string id)
 
         [HttpPost]
         public async Task<IActionResult> Create([FromBody] Volunteer item)
-        {
+        {
+            if (await dataInterface.BceidExistsAsync(item.Externaluseridentifier))
```

Comment:
I meant this call, should it be dataInterface.GetVolunteerByExternalId(item.Externaluseridentifier) like in the PUT handler?